colorizer_data/writer.py

```diff
@@ -54,7 +54,7 @@ class ColorizerDatasetWriter:
     more details.)
     """
 
-    outpath: Union[str, pathlib.Path]
+    outpath: str
```

Comment:
`outpath` was only ever a string value.

Reply:
> `outpath` was only ever a string value.

how do you know?

Reply:
It was set using `os.path.join` which returns a string type.